src/main/java/ch/admin/bar/siard2/cmd/PrimaryDataFromDb.java

```diff
@@ -211,7 +210,7 @@ private void getRecord(ResultSet rs, ch.admin.bar.siard2.api.Record record, Mime
         }
     }
 
-    private Cell getCell(ch.admin.bar.siard2.api.Record record, int cellIndex) throws IOException {
+    private Cell getCell(ch.admin.bar.siard2.api.TableRecord record, int cellIndex) throws IOException {
```

Comment:
Now you can replace this qualified import...

just add 
` import ch.admin.bar.siard2.api.TableRecord`
.

and the do `private Cell getCell(ableRecord record, int cellIndex) ...`

Reply:
<img width="974" height="308" alt="image" src="https://github.com/user-attachments/assets/d51aa541-890e-45d0-9407-4b81934cd539" />